geos-3.13.yaml

```
@@ -0,0 +1,126 @@
+package:
+  name: geos-3.13
+  version: 3.13.0
+  epoch: 0
+  description: GEOS is a library providing OpenGIS and JTS spatial operations in C++.
+  copyright:
+    - license: LGPL-2.1-or-later
+  dependencies:
+    provides:
+      - geos=${{package.full-version}}
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+\.\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cmake
+      - openssf-compiler-options
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/libgeos/geos
+      tag: ${{package.version}}
+      expected-commit: d7957246c588aa9c690efe67924fd70e741a06ab
+
+  - uses: cmake/configure
+
+  - uses: cmake/build
+
+  - uses: cmake/install
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-dev
+    pipeline:
+      - uses: split/dev
+    dependencies:
+      runtime:
+        - ${{package.name}}
+      provides:
+        - geos-dev=${{package.full-version}}
+    description: geos dev
```

```suggestion
    description: geos dev
    test:
      pipeline:
        - uses: test/pkgconf
```


Needs test/pkgconf here. I can fix if indentation breaks once I'm back to laptop. Please continue if you can add a subsequent commit here.